go.mod

```diff
@@ -76,6 +75,7 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20250218202821-56aae31c358a // indirect
 	google.golang.org/grpc v1.72.2 // indirect
 	google.golang.org/protobuf v1.36.6 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
Yeah, it will take time to fully get rid of it, since there are packages like jaypipes/ghw and gomega which import it.
Btw, great to at least start removing the direct calls 👍🏼

Reply:
Yep, I was trying to `replace` this in the go-mod but couldn't find a way forward there..